src/main/java/org/killbill/billing/plugin/adyen/client/model/PaymentInfo.java

```diff
@@ -218,7 +218,8 @@ public String getHouseNumberOrName() {
     }
 
     public void setHouseNumberOrName(final String houseNumberOrName) {
-        this.houseNumberOrName = houseNumberOrName;
+        // Adyen needs houseNumberOrName to not be blank in US https://docs.adyen.com/developers/api-reference/common-api/address
```

Comment:
Did you verify this is compatible with EMEA?

Reply:
yes it is compatible